pkg/annotations/annotations.go

```diff
@@ -114,6 +115,61 @@ func ParseAnnotations(annotations map[string]string, clusterName string) (*Annot
 	return a, nil
 }
 
+// ParseServiceAnnotations validates and loads all the annotations provided into the Annotations struct.
```

Comment:
This comment should mention it parses all annotations relevant to the service resource.

Also the `annotations.ParseAnnotations` should likely be refactored to `annotations.ParseIngressAnnotations` for consistency.

[or] you could combine these 2 functions as they're quite similar. Just needs to be smarter on which annotations to parse based on what's passed. I'm fine either way, as long as the intent of `annotations.ParseServiceAnnotations` vs `annotations.ParseAnnotations` is clear.